src/riak_kv_qry_compiler.erl

```diff
@@ -1911,7 +1933,7 @@ sum_sum_finalise_test() ->
       ).
 
 extract_stateful_function_1_test() ->
-    {ok, #riak_select_v1{ 'SELECT' = #riak_sel_clause_v1{ clause = [Select] } }} =
+    {ok, ?SQL_SELECT{ 'SELECT' = #riak_sel_clause_v1{ clause = [Select] } }} =
```

Comment:
Is this an upgrade test? If not, perhaps the record name should also be `?SQL_SELECT`